CB/dca-template-config.json

```diff
@@ -0,0 +1,16 @@
+{
+  "manifest_schemas": [
+    {
+      "display_name": "FileMapToIndividual",
+      "schema_name": "FileMapToIndividual",
+      "type": "records"
```

Comment:
Should be `record` instead of `records`. I will merge, but this manifests may not generate properly.